File: Day42/Java/check_Sorted_Array.java
public class check_Sorted_Array {
    static int isSorted(int n, int []a) {
        for(int i=1;i<n;i++){
            if(a[i]<a[i-1]){
                return 0;
            }
        }
        return 1;
    }
    public static void main(String[] args) {
        int []arr = {1,1,2,3,4,4,5,6,7,10};
        int ans = isSorted(arr.length, arr);
        System.out.println(ans);
    }
}

File: Day42/CPP/check_Sorted_Array.cpp
#include<bits/stdc++.h>
using namespace std;
int isSorted(int n, vector<int> a) {
    for(int i=1;i<n;i++){
        if(a[i]<a[i-1]) return false;
    }
    return true;
}

int main()
{
    int n;
    cin>>n;
    // vector<int>arr = {1, 2, 3 ,8 ,2 ,3 ,4 ,9};
    vector<int> arr = {1,1,2,3,4,4,6,7,10,20};
    int ans = isSorted(n,arr);
    cout<<ans;
    return 0;
}